.travis.yml

```diff
@@ -4,31 +4,16 @@ node_js:
   - "0.10"
   - "4"
   - "5"
-env:
-  - FIREFOX_VERSION=nightly
-#  - FIREFOX_VERSION=aurora
-#  - FIREFOX_VERSION=beta
-#  - FIREFOX_VERSION=release
-
+addons:
+  firefox: latest-beta
 notifications:
   irc: "irc.mozilla.org#jpm"
-
 before_install:
-  - "export DISPLAY=:99.0"
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
```

Comment:
It looks like that command originally came from documentation like this: https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI

There it lists the simplified version - `sh -e /etc/init/xvfb start` - and the complicated one, which seems to be necessary to set a specific screen resolution. I don't have any specific reason for simplifying this command.. I can replace the simplified version with that one.